react-geo-app/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@terrestris/react-geo": "^15.0.2",
+    "@terrestris/react-geo": "^16.2.0",
```

Comment:
Do we need to update (all) other libs as well?

Reply:
Not necessarily. I tested the current code base and everything looks good. However, users that will follow the workshop will now use webpack 5 (via create-react-app). So their versions will diverge from the ones specified here. But that will always be the case, anyways.